AudioToTxt/source/AudioToTxt.cpp

```
@@ -107,5 +107,63 @@ namespace Captions{
     
         return "";
     }
+    CaptionCache::CaptionCache(const std::string& caption_srt){
+        loadCaptions(caption_srt);
+    }
+
+
+    void CaptionCache::loadCaptions(const std::string& caption_srt){
+        std::ifstream file(caption_srt);
+        if (!file){
+            std::cerr << "Error: Could not open file: " << caption_srt << std::endl;
```

Should probably throw an error, but this works.